docs/hellonode.md

```diff
@@ -133,74 +133,114 @@ A kubernetes **pod** is a group of containers, tied together for the purposes of
 Create a pod with the `kubectl run` command:
 
 ```shell
-kubectl run hello-node \
+$ kubectl run hello-node \
     --image=gcr.io/PROJECT_ID/hello-node:v1 \
     --port=8080
-CONTROLLER  CONTAINER(S)  IMAGE(S)                     SELECTOR        REPLICAS
-hello-node  hello-node    gcr.io/..../hello-node:v1    run=hello-node  1
+deployment "hello-node" created
 ```
 
-Now is probably a good time to run through some of the following interesting kubectl commands (none of these will change the state of the cluster, full documentation is available [here](https://cloud.google.com/container-engine/docs/kubectl/)):
+As shown in the output, the `kubectl run` created a **deployment** object.  Deployments are the recommended way for managing creation and scaling of pods.  In this example, a new deployment manages a single pod replica running the *hello-node:v1* image.
+
+To view the deployment we just created:
+
+```shell
+$ kubectl get deployments
+NAME         DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+hello-node   1         1         1            1           3m
+```
+
+To view the pod created by the deployment:
 
 ```shell
 $ kubectl get pods
-$ kubectl logs
-$ kubectl cluster-info
-$ kubectl config view
+NAME                         READY     STATUS    RESTARTS   AGE
+hello-node-714049816-ztzrb   1/1       Running   0          6m
+```
+
+To view the stdout / stderr from a pod (hello-node image has no output, so logs will be empty in this case):
+
+```shell
+$ kubectl logs <POD-NAME>
+```
+
+To view metadata about the cluster:
+
+```shell
+$ kubectl get cluster-info
+```
+
+To view cluster events:
+
+```shell
 $ kubectl get events
 ```
 
+Full documentation for kubectl commands is available [here](https://cloud.google.com/container-engine/docs/kubectl/)):
+
 At this point you should have our container running under the control of Kubernetes but we still have to make it accessible to the outside world.
 
 ## Allow external traffic
 
-By default, the pod is only accessible by its internal IP within the Kubernetes cluster. In order to make the `hello-node` container accessible from outside the kubernetes virtual network, you have to expose the pod as a kubernetes service.
+By default, the pod is only accessible by its internal IP within the Kubernetes cluster. In order to make the `hello-node` container accessible from outside the kubernetes virtual network, you have to expose the pod as a kubernetes **service**.
```

Comment:
I've addressed this by linking the first reference to the docs.  Updated references for pod and deployment to link as well.